plio/cache.py

```diff
@@ -0,0 +1,34 @@
+from django.core.cache import cache
+
+
+def get_cache_key(instance):
+    """Calculates cache key for an instance"""
+    instance_class = instance.__class__.__name__
+    return {
```

Comment:
First time I'm seeing `django-redis`, so maybe this is a dumb question. But is this same method used for all cache hits? So if it's a plio, then other keys will be empty and plio will be the one with a value?

Reply:
Yes, it's switch/case functionality. So it matches the plio class with the keys and returns the corresponding value (which is the cache key name).

For plio instance ID:1, it will return plio_1
For video instance ID:99, it will return video_99